cli_tools/gce_onestep_image_import/onestep_importer/utils.go

```diff
@@ -0,0 +1,183 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/flags"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"github.com/dustin/go-humanize"
+)
+
+// runCmd executes the named program with given arguments.
+// Logs the running command.
+func runCmd(name string, args []string) error {
+	log.Printf("Running command: '%s %s'", name, strings.Join(args, " "))
+	cmd := exec.Command(name, args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+// handleTimeout signals caller by closing the timeout channel after the
+// specified time has elapsed.
+func handleTimeout(timeout time.Duration, timeoutChan chan struct{}) {
+	time.Sleep(timeout)
```

Comment:
I think the 3min should be deducted here. Not only ImageImport but also export/transferFile should also reserve 3min for cancel/cleanup.

Reply:
done